LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Steven Berlanga
+Copyright (c) 2015 Steven Berlanga, Google Inc.
```

Comment:
This came in on through our outbound open source process. I'll check for the reason.